examples/automm/Kaggle-Pawpularity/README.md

```diff
@@ -0,0 +1,103 @@
+﻿Take [Petfinder Pawpularity competition](https://www.kaggle.com/competitions/petfinder-pawpularity-score/overview) as an example showing how to use AutoMMPredictor to complete a competition.
+
+## 1. Kaggle Kernel-only Competition with AutoGluon
+
+In a kaggle competition, especially a code competition, users cannot obtain AutoGluon resources through the network. 
+To solve the problem, there are two key points:
+
+ - Get AutoGluon and its dependent libraries.
+ - Save the downloaded resources when saving your model.
+
+### 1.1 Get AluoGluon in Kaggle
```

Comment:
For this part, need to to be more clear what should we do to "upload the installed AutoGluon" to Kaggle.